src/Conversion/KrnlToLLVM/ConvertKrnlToLLVM.cpp

```diff
@@ -520,12 +520,13 @@ void ConvertKrnlToLLVMPass::runOnOperation() {
   target.addLegalOp<ModuleOp>();
   target.addLegalOp<UnrealizedConversionCastOp>();
 
+  // Get active accelerators if any.
+  SmallVector<onnx_mlir::accel::Accelerator *, 2> activeAccels;
```

Comment:
Out of curiosity, would it be sufficient to return a reference to the class list with a const?
`const SmallVectorImpl<onnx_mlir::accel::Accelerator>& getActiveAccelerator()`

Reply:
Removed this function since it becomes redundant now.